weldx/asdf/util.py

```diff
@@ -112,6 +113,32 @@ def _write_read_buffer(
     return _read_buffer(buffer, open_kwargs)
 
 
+def get_yaml_header(file) -> str:
```

Comment:
Type hints for `file`? What about `Path`?

Reply:
everything that works with `read` should work :)
I will leave it to the docstring for now